planning/obstacle_avoidance_planner/src/mpt_optimizer.cpp

```diff
@@ -633,25 +643,8 @@ MPTOptimizer::ValueMatrix MPTOptimizer::generateValueMatrix(
   Eigen::SparseMatrix<double> Qex_sparse_mat(D_x * N_ref, D_x * N_ref);
   std::vector<Eigen::Triplet<double>> Qex_triplet_vec;
   for (size_t i = 0; i < N_ref; ++i) {
-    // this is for plan_from_ego
-    const bool near_kinematic_state_while_planning_from_ego = [&]() {
-      const size_t min_idx = static_cast<size_t>(std::max(0, static_cast<int>(i) - 20));
-      const size_t max_idx = std::min(ref_points.size() - 1, i + 20);
-      for (size_t j = min_idx; j <= max_idx; ++j) {
-        if (ref_points.at(j).plan_from_ego && ref_points.at(j).fix_kinematic_state) {
-          return true;
-        }
-      }
-      return false;
-    }();
```

Comment:
Do not remove this please, should be commented out instead.

Reply:
add it again as coment. https://github.com/autowarefoundation/autoware.universe/pull/1191/commits/7b8fb4febd4e73938894e2b0c494d6715ded9e68